nbs/sidebar.yml

```diff
@@ -2,3 +2,4 @@ website:
   sidebar:
```

Comment:
_Commented on notebook `nbs/sidebar.yml` Cell 1 From line 4 to 5_
```python
      - index.ipynb
      - example.ipynb
```
A new multi-line comment

_Respond and view the context [here](https://gitnotebooks.com/GitNotebooks/docs/pull/2)._